crates/indexer/src/cli.rs

```diff
@@ -0,0 +1,149 @@
+use anyhow::Context;
+use clap::Parser;
+use futures::StreamExt;
+use hex_literal::hex;
+use std::{cmp::Ordering, error::Error, str::FromStr, vec};
+mod stream;
+use apibara_client_protos::pb::starknet::v1alpha2::{Filter, HeaderFilter};
+use log::{debug, info, warn};
+use prisma_client_rust::bigdecimal::{
+    num_bigint::{BigUint, ToBigUint},
+    Num,
+};
+use tokio::sync::mpsc;
+mod prisma;
+
+/// Command line args parser.
+/// Exits with 0/1 if the input is formatted correctly/incorrectly.
+#[derive(Parser, Debug)]
+#[clap(version, verbatim_doc_comment)]
+struct Args {
+    /// The world to index
+    world: String,
+    /// The RPC endpoint to use
+    rpc: String,
+}
+
+#[tokio::main]
+async fn main() -> anyhow::Result<()> {
+    let args = Args::parse();
+
+    let world = BigUint::from_str_radix(&args.world[2..], 16).unwrap();
```

Comment:
Lets provide some feedback on errors for user provided inputs

For example
```suggestion
    let world = BigUint::from_str_radix(&args.world[2..], 16).unwrap_or_else(|error| {
        panic!("Failed parsing world address: {error:?}");
    });
```